guides/use-microprofile-amqp-connector-with-ssl-openshift.adoc

```diff
@@ -0,0 +1,208 @@
+
+Run following command to deploy AMQ 7 broker with secured acceptor:
+[source, shell]
+----
+oc create -f broker.yaml
+----
+
+Check that AMQ 7 broker is in `Running` state by checking running pods:
+```[source, bash ,options="nowrap"]
+$ oc get pods
+NAME                                            READY   STATUS      RESTARTS   AGE
+amq-broker-ss-0                                 1/1     Running     0          79m
+...
+```
+
+== Build Reactive Messaging application using WildFly Glow
+
+We will use an example of Reactive Messaging application in this guide, that consists of a single `ProducingBean` and `ConsumingBean`.
+Where `ProducingBean` periodically produces messages into the channel and `ConsumingBean` consumes them. It also contains needed scripts and yaml files to set up AMQ 7 broker on OpenShift.
+
+Example application is available on GitHub:
+[source, bash]
+----
+git clone git@github.com:wildfly-extras/guides.git
+cd guides/microprofile-reactive-messaging-amqp-connector-example/
+----
+
+The important part of the application is `microprofile-config.properties` file configuring secured AMQP connector to remote AMQ 7 broker:
+[source, bash, options="nowrap"]
+----
+amqp-host=${AMQ_HOST}
+amqp-port=443
+amqp-username=admin
+amqp-password=admin
+amqp-use-ssl=true
+mp.messaging.connector.smallrye-amqp.wildfly.elytron.ssl.context=amqp-ssl-context
+
+mp.messaging.outgoing.source.connector=smallrye-amqp
+
+mp.messaging.incoming.in.connector=smallrye-amqp
+mp.messaging.incoming.in.address=source
+----
+
+`amqp-host` - url of remote AMQ 7 broker deployed on OpenShift, for example "amq-broker-all-0-svc-rte-mnovak.apps.mnovak-oegn.psi.domain.com".
+
+`amqp-port` - port where AMQ 7 broker is listening, in case of OpenShift there is used port `443` for secured communication
+
+`amqp-username` and `amqp-password` - username and password for authentication to AMQ 7 broker. `admin/admin` is default used in AMQ 7 broker
+
+`amqp-use-ssl=true` - specifies that we want to use a secure connection when connecting to the broker.
+
+`mp.messaging.outgoing.source.connector=amqp-ssl-context` - this is not needed if there are CA signed certificates. However, in our case we're using self-signed certificates, so we need to specify a truststore
+in the Elytron subsystem and create an SSLContext referencing that. The value of this property is used to look up the SSLContext in the Elytron subsystem under /subsystem=elytron/client-ssl-context=* in the WildFly management model.
+In this case the property value is `amqp-ssl-context`, so we look up the SSLContext defined by /subsystem=elytron/client-ssl-context=amqp-ssl-context and use that to configure the truststore to use for the connection to AMQ broker.
+For this reason there is defined CLI script `scripts/config.cli` which will be used to configure client-ssl-context during build of trimmed WildFly server:
```

Comment:
s/during build of trimmed WildFly server/when the WildFly server is provisioned and configured.